docs-src/concepts/what-is-an-event.md

```diff
@@ -13,3 +13,23 @@ Events are created by the Temporal Cluster in response to external occurrences a
 All Events are recorded in the [Event History](/concepts/what-is-an-event-history).
 
 A list of all possible Events that could appear in a Workflow Execution Event History is provided in the [Event reference](/references/events).
+
+#### Activity Events
+
+There are seven Activity-related Events that are added to History at different points in an Activity Execution:
+
+- After a [Workflow Task Execution](/concepts/what-is-an-activity-task-execution) reaches a line of code that starts/executes an Activity, the Worker sends the Activity type and arguments to the Cluster, and the Cluster adds an [ActivityTaskScheduled](/references/events#activitytaskscheduled) Event to History.
+- When ActivityTaskScheduled is added to History, the Cluster adds a corresponding Activity Task to the Task Queue.
```

Comment:
```suggestion
- After a [Workflow Task Execution](/concepts/what-is-an-activity-task-execution) reaches a line of code that starts/executes an Activity, the Worker sends the Activity Type and arguments to the Temporal Cluster, and the Cluster adds an [ActivityTaskScheduled](/references/events#activitytaskscheduled) Event to Event History.
- When `ActivityTaskScheduled` is added to History, the Cluster adds a corresponding Activity Task to the Task Queue.
```

Reply:
why use code here and not when linking?

Reply:
Combining monospace and link styling can create reading difficulties, so the next version of the documentation style guide will include this guideline.

Reply:
Have you looked into changing the styling?

Reply:
Def something to consider.